### PR TITLE
Add a comment saying where distortion_model.hpp originally came from.

### PR DESCRIPTION
**Pull request by clalancette**
Signed-off-by: Chris Lalancette <clalancette@osrfoundation.org>

### CODE REVIEW

**Comment by clalancette** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/2602/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/183/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/2060/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/2719/)

File: sensor_msgs/include/sensor_msgs/distortion_models.hpp [hunk 1]
```diff
@@ -31,6 +31,9 @@
 // ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
+// This file is originally ported from ROS1:
```
Comment (mikaelarguedas) [inline sensor_msgs/include/sensor_msgs/distortion_models.hpp]: not sure "ported" applies here, there is nothing ROS (1 or 2) specific in this file

Comment (clalancette) [inline sensor_msgs/include/sensor_msgs/distortion_models.hpp]: True, but the original file is from a ROS1 repository.  Should I change it to just:

"This file is originally from:"

?

Comment (mikaelarguedas) [inline sensor_msgs/include/sensor_msgs/distortion_models.hpp]: yes or "this file was originally copied from". Not sure about the exact wording but I think a more generic phrasing will allow for more reuse throughout the codebase

**Comment by clalancette** [comment]
I changed the wording in all three places it was used before.  New CI:

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/2604/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/184/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/2061/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/2720/)

**Comment by dirk-thomas** [comment]
Do we need CI builds for a comment-only change?

**Comment by clalancette** [comment]
Heh, the last time I didn't do one, I ended up screwing things up (the linters can still fire off for comment-only changes).  So I got conservative and ended up doing it for everything.

**Comment by mikaelarguedas** [comment]
I think it's good to make sure not to break the tests by modifying comment sections. Then I agree that running on all platforms and all tests is overkill. A single platform (Windows is usually the pickiest) testing only the modified package would be more sensible IMO.

File: sensor_msgs/include/sensor_msgs/distortion_models.hpp [hunk 2]
```diff
@@ -31,6 +31,9 @@
 // ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
+// This file is originally from:
+// https://github.com/ros/common_msgs/blob/jade-devel/sensor_msgs/include/sensor_msgs/distortion_models.h
```
Comment (dirk-thomas) [inline sensor_msgs/include/sensor_msgs/distortion_models.hpp]: Please include the exact hash when this file was forked.

**Comment by clalancette** [comment]
I ended up encoding the hashes into the URLs themselves.  That way we know exactly from whence it came.

**Pull request merged**